.travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: enabled
```

Comment:
that config option is ignored by travis; sudo is enabled by default

Reply:
Good to know.  I removed it.